src/axios/index.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+// eslint-disable-next-line import/no-unresolved
```

Comment:
let's try to remove need of this line by extending eslint config. try this solution:
https://stackoverflow.com/questions/41769880/how-to-manually-add-a-path-to-be-resolved-in-eslintrc